source/tutorials/pnp-testsuites/pnp-testsuites.rst

```diff
@@ -0,0 +1,545 @@
+
+    #. Set the encoder binary file executable, and execute the encoder sample binary file to get FPS.
+       Since the encoder program may not fully utilize 100% of the eecoding engine's capability
+       on many platform, it's recommended to create a script to run multiple (e.g. 4) instances
+       of the encoding program in parallel, and aggregate the reported FPS numbers
+       as the final result.
+
+Video Conference - 1080p30 WebRTC
+---------------------------------
+
+This KPI measures the smoothness of WebRTC using HSC.
+
+**Environment Setup**
+
+    #. Device is maintained in a stable state and is not running any other applications.
+
+    #. Set up HSC environment.
+
+**Measurement steps**
+
+    #. Open the browser with the URL :file:`https://apprtc.appspot.com`
+
+    #. Click :kbd:`Join` to enter a room.
+
+    #. Calculate FPS using HSC.
+
+GFXBench 5.0 Manhattan 3.1 Benchmark Off Screen
+-----------------------------------------------
+
+This KPI measures the graphics performance using `GFXBench <https://gfxbench.com/>`_ application.
+
+**Environment Setup**
+
+    #. Device is maintained in a stable state and is not running any other applications.
+
+**Measurement Steps**
+
+    #. Download the GFXBench benchmark application from the Google* Play* store and install it.
+
+    #. Launch the GFXBench GL benchmark application, select :kbd:`Manhattan 3.1 Benchmark Off Screen` to run.
+
+    #. Get the result.
+
+Geekbench* Multi Core
+---------------------
+
+This KPI measures the processor and memory performance using the
+`Geekbench <https://www.geekbench.com/>`_ application.
+
+**Environment Setup**
+
+    #. Device is maintained in a stable state and is not running any other applications.
+
+**Measurement Steps**
+
+    #. Download the Geekbench app from the Google Play store and install it.
+
+    #. Launch the Geekbench app and run it.
+
+    #. Get the result.
+
+Antutu* v6.3.3
+--------------
+
+This KPI measures system performance using the `Antutu <https://www.antutu.com/>`_ application.
+
+**Environment Setup**
+
+    #. Device is maintained in a stable state and is not running any other applications.
+
+**Measurement Steps**
+
+    #. Download the Antutu application from
+       `Antutu6.3.3 <https://antutu-benchmark.en.uptodown.com/android/download/1462551>`_ and install it.
+
+    #. Launch the Antutu app and start testing.
+
+    #. Record score.
+
+Browser Warm Launch Latency
+---------------------------
+
+This KPI measures the warm launch time of a browser using HSC.
+
+**Environment Setup**
+
+    #. Device is maintained in a stable state and is not running any other applications.
+
+    #. Open the browser app and nevigate to the example testing website.
+       Press :kbd:`home` to move the app to the background, and then launch
+       the browser app again to check the original nevigation can be comlete
+       automatically without manual operations (i.e. warm launch).
```

Comment:
Change "nevigate" to "navigate"
Change "nevigation can be comlete" to "navigation can be complete"